packages/web-components/fast-foundation/src/tree-item/tree-item.template.ts

```diff
@@ -17,28 +18,27 @@ export function treeItemTemplate<T extends FASTTreeItem>(
             tabindex="-1"
             class="${x =>
                 [
+                    x.disabled && "disabled",
                     x.expanded && "expanded",
-                    x.selected && "selected",
                     x.nested && "nested",
-                    x.disabled && "disabled",
+                    x.selected && "selected",
                 ]
                     .filter(Boolean)
                     .join(" ")}"
-            aria-expanded="${x =>
-                x.childItems && x.childItemLength() > 0 ? x.expanded : void 0}"
-            aria-selected="${x => x.selected}"
-            aria-disabled="${x => x.disabled}"
+            aria-expanded="${x => x.ariaExpanded}"
```

Comment:
This is actually intentional - if there are no children, its not *actually* expandable. Setting this to false when there are no children means that it *is* expandable, which its not.